### ISSUE TITLE
Initialize Filter Value

### ISSUE THREAD

**Issue by frank1340** [aslagle/reactive-table#347]
Hello, 

I have about 7 filters on a table.

I want to set up an initial filter condition for one or two fields when a session begins.

After that, I want the natural behavior of Reactive Table to persist while also maintaining the value of the input field until the session ends for all fields.

I've made this work using onRendered, but I have duplicative code, setting the filter when rendered & when the select value changes.

Is this the correct way to accomplish my goals  or am I am missing a better, simpler solution ?  Thank you.

**Comment by frank1340** [comment]
Along with that, what if the filter is reliant upon a collection loading, currently autopublish or just publish?

**Comment by aslagle** [comment]
Using onRendered sounds fine. If you have too much duplicated code, define a function that sets the filters somewhere else and call it from both places. 

You might want to try something like Template.subscriptionsReady to wait for the collection to load before setting filters.

**Comment by frank1340** [comment]
What about current page?  The user is sent to the first page of results because of the results being re-run.  It conflicts with the onRendered code.  Am I missing something?

**Comment by aslagle** [comment]
Setting a filter resets the current page, so you need to set current page after you set the filters. You can use a ReactiveVar for the current page so you can get and set it.

**Comment by frank1340** [comment]
Yeah, I get the ordering now, I will go with that.  Thanks for the help.
